### PR TITLE
[14.0][MIG] partner_label

### PR DESCRIPTION
**Pull request by dnplkndll**
just basic migration, I have a US Letter 30 up report that would be nice to add here next. 
@wlin-kencove does it look like our ken_crm_reports/views/label_report.xml would be possible to add here and select on the company settings as the label format? would need to parameterize the size more but that seems doable.

### CODE REVIEW

**Comment by newtratip** [comment]
@dnplkndll, I will close my pr https://github.com/OCA/partner-contact/pull/1040

Please change the title to `[14.0][MIG] partner_label` for easy search with the module name in Pull requests.

File: partner_label/reports/res_partner.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                >
+                    <address
+                        t-field="this.self"
+                        t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'
+                    />
+                </div>
+            </t>
+        </t>
+    </template>
+    <report
```
Comment (newtratip) [inline partner_label/reports/res_partner.xml]: report tag is deprecate https://github.com/odoo/odoo/pull/54564 please change to
```suggestion
    <record id="report_res_partner_label" model="ir.actions.report">
        <field name="name">Partner Labels</field>
        <field name="model">res.partner</field>
        <field name="report_type">qweb-pdf</field>
        <field name="report_name">partner_label.view_res_partner_label</field>
        <field name="binding_model_id" ref="base.model_res_partner" />
        <field name="binding_type">report</field>
        <field name="paperformat_id" ref="base.paperformat_euro" />
    </record>
```

**Comment by dnplkndll** [comment]
looks like we are all green @newtratip thanks

**Comment by dnplkndll** [comment]
@max3903 Here is a module with some redundancy on our catalog request mailing labels. can review if we could add the website signup to a pdf from marketing assistant to mail them the next morning. we have a module with the requester (bot/mass mail or user),  product (track usage), requested date, sent date and the utm source/camp.

**Comment by alexeirivera87** [comment]
Hello
This PR is going to be continued? will it be merged to 14.0?

Regards

**Comment by NL66278** [comment]
Duplicate with #1095

**Pull request closed without merging**